hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -789,4 +791,31 @@ public static String getOMAddressListPrintString(List<OMNodeDetails> omList) {
     printString.append("]");
     return printString.toString();
   }
+
+  public static String format(List<ServiceInfo> nodes, int port,
+                              String leaderId) {
+    StringBuilder sb = new StringBuilder();
+    int count = 0;
+    for (ServiceInfo info : nodes) {
+      // Printing only the OM's running
+      String role = info.getHostname().equals(leaderId) ? "LEADER" : "FOLLOWER";
```

Comment:
Shouldn't we compare `info.getOmRoleInfo().getNodeId()` to `leaderId`?  If hostname and node ID are different, currently we get FOLLOWER for all 3 OMs.
<img width="1643" alt="Screenshot 2022-07-25 at 14 09 40" src="https://user-images.githubusercontent.com/6454655/180774549-265d5c53-c0b7-48e3-8d61-16673b880aeb.png">